.github/workflows/build_and_run.yaml

```diff
@@ -24,9 +35,21 @@ jobs:
       - name: Generate initial data for training
         working-directory: ./GameServers/VSharp/VSharp.ML.GameServer.Runner/bin/Release/net7.0
         run: |
-          dotnet VSharp.ML.GameServer.Runner.dll --mode generator --datasetbasepath ../../../../../../maps/DotNet/Maps/Root/bin/Release/net7.0 --datasetdescription ../../../../../../maps/DotNet/Maps/dataset_for_tests.json --stepstoserialize 200
+          dotnet VSharp.ML.GameServer.Runner.dll --mode generator --datasetbasepath ../../../../../../maps/DotNet/Maps/Root/bin/Release/net7.0 --datasetdescription ../../../../../../workflow/dataset_for_tests.json --stepstoserialize 200
+
+      - name: Move SerializedEpisodes
+        working-directory: ./AIAgent
+        run: |
+          mkdir report
+          mv ../GameServers/VSharp/VSharp.ML.GameServer.Runner/bin/Release/net7.0/8100/SerializedEpisodes/ report/
 
       - name: Launch application
+        working-directory: ./AIAgent
+        run: |
+          python3 launch_servers.py &
```

Comment:
Looks like ```launch_servers``` also should be configurable. At least, one can specify number of servers.

Reply:
Lets do it in another issue

Reply:
mentioned in #4